docs/en/v0.4/user-guide/monitoring.md

```diff
@@ -0,0 +1,136 @@
+  --set prometheusMonitor.labelsSelector.release="prometheus"
+```
+
+## Metrics Detail
+
+| Key                                        | Description                                            | Type    |
+|--------------------------------------------|--------------------------------------------------------|---------|
+| greptime_opendal_requests_total            |                                                        | counter |
+| greptime_meta_create_catalog               |                                                        | counter |
+| greptime_meta_create_schema                |                                                        | counter |
+| greptime_opendal_bytes_total               |                                                        | counter |
+| greptime_process_virtual_memory_max_bytes  | Maximum amount of virtual memory available in bytes.   | gauge   |
+| greptime_catalog_schema_count              |                                                        | gauge   |
+| greptime_catalog_catalog_count             |                                                        | gauge   |
+| greptime_process_start_time_seconds        | Start time of the process since unix epoch in seconds. | gauge   |
+| greptime_process_threads                   | Numberof OS threads in the process.                    | gauge   |
+| greptime_process_cpu_seconds_total         | Total user and system CPU time spent in seconds.       | gauge   |
+| greptime_process_virtual_memory_bytes      | Virtual memory size in bytes.                          | gauge   |
+| greptime_sys_jemalloc_resident             |                                                        | gauge   |
+| greptime_sys_jemalloc_allocated            |                                                        | gauge   |
+| greptime_process_max_fds                   | Maximum number of open file descriptors.               | gauge   |
+| greptime_app_version                       |                                                        | gauge   |
+| greptime_runtime_threads_idle              |                                                        | gauge   |
+| greptime_runtime_threads_alive             |                                                        | gauge   |
+| greptime_process_resident_memory_bytes     | Resident memory size in bytes.                         | gauge   |
+| greptime_process_open_fds                  | Number of open file descriptors.                       | gauge   |
+| greptime_opendal_requests_duration_seconds |                                                        | summary |
+| greptime_meta_create_catalog               |                                                        | summary |
+| greptime_http_track_metrics                |                                                        | summary |
+| greptime_meta_create_schema                |                                                        | summary |
```

Comment:
I assume this is not the full set of our metrics. We can state this fact in docs and list only key/informative metrics here.

Reply:
I see. I believe there are more if you create some traffic. Metrics are created lazily. It might be a good idea to extract them from our cloud instances.

Reply:
Later I will select some metrics from our cloud.

Reply:
I choose some metrics in detail，it is necessary add or delete ?